From c8ac83fed152b0c7bebaea30237a0cd2fe054f1b Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 25 Apr 2019 14:12:47 +0700
Subject: [PATCH 060/328] arm64: dts: renesas: cat875: enable swapping odd/even
 data for dual channels LVDS

In sub board, pin-outs of LVDS-IF to dual channels LVDS panel are
described in Schematics:
- LVDS0 : Even Data.
- LVDS1 : Odd Data.

Dual channels LVDS panel will use LVDS1 for 1st pixel and LVDS0 for 2nd
pixel. It is reversed with default register.
So that to support right channel, add "swap" property in LVDS nodes
in devicetree.

This patch does this thing.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/cat875.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/cat875.dtsi b/arch/arm64/boot/dts/renesas/cat875.dtsi
index 6d00e7d4..10ac3a6 100644
--- a/arch/arm64/boot/dts/renesas/cat875.dtsi
+++ b/arch/arm64/boot/dts/renesas/cat875.dtsi
@@ -126,6 +126,7 @@
 		 <&x13_clk>;
 	clock-names = "fck", "dclkin.0";
 	mode = "dual-link";
+	swap = <1>;
 
 	ports {
 		port@1 {
@@ -143,6 +144,7 @@
 		 <&x13_clk>;
 	clock-names = "fck", "dclkin.0";
 	mode = "dual-link";
+	swap = <1>;
 
 	ports {
 		port@1 {
-- 
1.9.1

